lua/llm/language_server.lua

```diff
@@ -185,15 +181,17 @@ function M.setup()
     return
   end
 
-  local cmd
```

Comment:
I'd leave this here and put `cmd = { bin_path }` in an else clause after line 200, for reading consistency.

Reply:
ty for review. I've pushed a fixup tested locally. If it's good I can squash or you can do it from github UI.